Src/FluentAssertions/Equivalency/Steps/GenericDictionaryEquivalencyStep.cs

```diff
@@ -76,8 +76,9 @@ private static bool AssertSubjectIsNotNull(object subject)
         if (actualDictionary is null)
         {
             AssertionScope.Current.FailWith(
-                $"Expected {{context:subject}} to be a dictionary or collection of key-value pairs that is keyed to type {expectedDictionary.Key}. " +
-                $"It implements {actualDictionary}.");
+                "Expected {context:subject} to be a dictionary or collection of key-value pairs that is keyed to " +
+                $"type {expectedDictionary.Key}. " +
+                "It implements <null>.");
```

Comment:
🔧 The commit message is wrong. This code will never be executed if the subject is non-generic.